src/docs/framework-integration/angular.md

```diff
@@ -157,9 +200,11 @@ yarn link name-of-your-stencil-package
 
 The name of your Stencil package should match the `name` property from the Stencil component library's `package.json`.
 
-Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the changes to the Angular component library.
+Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the
+changes to the Angular component library.
 
-> **NOTE**: As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy, however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+> **NOTE**: As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy,
+> however, will modify your `package.json` so it is important to make sure you do not commit those changes.
```

Comment:
maybe this is very obvious if you're an angular-er, but it might be nice to include a final step which just includes whatever commands are necessary to get the angular dev server running with the stencil components built and included, so someone following this tutorial can assure themselves that they've created something that works correctly

Reply:
I did `npm run watch` in the top-level of the angular workspace and that seems to have done what I want here, I think?

Reply:
Hmmm. We could. That just starts to get outside the bounds of Stencil and the wrapper and more into consuming the proxies in an Angular app. Not sure how far we wanna go down that path. Guess it depends who we're targeting with these docs: developers familiar with the framework being targeted, or developers who may have little to no experience with these frameworks. 

I'd argue that once we get outside the bounds of the wrappers, there are plenty of other docs available for getting a consuming app up and running. We could potentially link out to some resource like that if we want. Thoughts? @rwaskiewicz @alicewriteswrongs

Reply:
I do think there's a gap between this section and https://github.com/ionic-team/stencil-site/pull/951/files#diff-22dd5394aaba28bc0132da1cf756f97ec72e56ef3c64cab0c0bd0471550bc21eR209 (that existed prior to this commit).  While I agree we don't want to get into the business of describing every way to integrate with Angular in the docs, I do agree with @alicewriteswrongs that we should have a way to verify that each framework wrapper set up works. This is a decent sized pain point for new users, and I think is something we can improve on. I don't want us to get into the business of providing support for every Angular/React/Vue use case, but do think we should provide enough technical detail to say "Here is what you're doing and why. Here is an example that builds on that. Exercises in getting this working in other architectures are left to the reader (paraphrased)

Reply:
^^ my thoughts exactly, basically just after following the steps I was left wondering "did that all work?" and it is always good to leave a tutorial knowing you have something working to build on

Reply:
That "consumer usage" section still exists as it did before this re-work. But, I get where you guys are coming from. So, my plan is to add on to the "consumer usage" and add an MVP for creating an Angular app and ingesting the wrappers. I'll also see if anything like this exists for the other wrappers and add to those if needed as well